src/pineko/theory.py

```diff
@@ -15,11 +15,21 @@
 import rich
 import yaml
 
-from . import check, configs, evolve, parser, theory_card
+from . import check, configs, evolve, parser, scale_variations, theory_card
 
 logger = logging.getLogger(__name__)
 
 
+def check_scvar_evolve(grid, max_as, max_al, tocheck):
+    """Check if the scale variations and central orders are to enough to construct the fktable."""
+    checkres, max_as_effective = check.contains_sv(grid, max_as, max_al, tocheck)
```

Comment:
```suggestion
    available, max_as_effective = check.contains_sv(grid, max_as, max_al, kind)
```

`checkres` is a bit generic. In case you want it, just use `res`, since `check` is in the name of the function called, and not ambiguous in this context.

Reply:
[e76bd60](https://github.com/NNPDF/pineko/pull/75/commits/e76bd60d64bb9be1dedd3b4230fd9453401cf10b)